boring/src/ssl/mod.rs

```diff
@@ -621,6 +632,15 @@ impl SslCurve {
     pub const SECP521R1: SslCurve = SslCurve(ffi::NID_secp521r1);
 
     pub const X25519: SslCurve = SslCurve(ffi::NID_X25519);
+
+    #[cfg(feature = "post-quantum")]
+    pub const X25519_KYBER512_DRAFT00: SslCurve = SslCurve(ffi::NID_X25519Kyber512Draft00);
+
+    #[cfg(feature = "post-quantum")]
+    pub const P256_KYBER768_DRAFT00: SslCurve = SslCurve(ffi::NID_P256Kyber768Draft00);
+
+    #[cfg(feature = "post-quantum")]
+    pub const X25519_KYBER768_DRAFT00: SslCurve = SslCurve(ffi::NID_X25519Kyber768Draft00);
```

Comment:
There is also `NID_X25519Kyber768Draft00Old`, the old codepoint for X25519Kyber768Draft00.